### PR TITLE
fix ftp.rb 

### PR DESCRIPTION
**Pull request by dacaiguoguo**
add
 connection.upload! 'info.txt'
info.txt contains some info of project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,19 @@ $ ipa build
 $ ipa distribute
 ```
 
+<<<<<<< HEAD
```
Comment (KrauseFx) [inline README.md]: Looks like you have a merge conflict here

**Comment by dacaiguoguo** [comment]
Watch OS 2 will fail 
```
target specifies product type 'com.apple.product-type.watchkit2-extension', but there's no such product type for the 'iphoneos' platform
```
so I delete -sdk option in flags
```
flags << %{-sdk #{@sdk}}
```

**Comment by dacaiguoguo** [comment]
@KrauseFx can you review my fix?

**Comment by KrauseFx** [comment]
Hey @dacaiguoguo, sorry I don't have commit access to this repo, maybe @kattrali can look into this?

**Pull request closed without merging**